### PR TITLE
Secret path from env

### PR DESCRIPTION
**Pull request by dekiel**
**Description**

Github issue creator and finder services read github token from path provided in environment variable.

### CODE REVIEW

File: development/gcp/cloud-run/search-github-issue/search-github-issues.go [hunk 1]
```diff
@@ -56,6 +55,7 @@ func main() {
 	port = os.Getenv("LISTEN_PORT")
 	githubOrg = os.Getenv("GITHUB_ORG")
 	githubRepo = os.Getenv("GITHUB_REPO")
+	toolsGithubTokenPath = os.Getenv("TOOLS_SAP_TOKEN_PATH")
```
Comment (Ressetkk) [inline development/gcp/cloud-run/search-github-issue/search-github-issues.go:58]: please keep the name consistency:
ENV_NAME => envName
```suggestion
	port = os.Getenv("PORT")
	githubOrg = os.Getenv("GITHUB_ORG")
	githubRepo = os.Getenv("GITHUB_REPO")
	toolsGithubTokenPath = os.Getenv("TOOLS_GITHUB_TOKEN_PATH")
```

Or find different name for the env name. Let's try to keep the names short as possible. Maybe `GHE_TOKEN_PATH`, which translates to "GitHub Enterprise Token Path".

File: development/gcp/cloud-run/create-github-issue/create-github-issue.go [hunk 2]
```diff
@@ -51,6 +52,7 @@ func main() {
 	port = os.Getenv("LISTEN_PORT")
 	githubOrg = os.Getenv("GITHUB_ORG")
 	githubRepo = os.Getenv("GITHUB_REPO")
+	toolsGithubTokenPath = os.Getenv("TOOLS_SAP_TOKEN_PATH")
```
Comment (Ressetkk) [inline development/gcp/cloud-run/create-github-issue/create-github-issue.go:55]: please keep the name consistency:
ENV_NAME => envName
```suggestion
	port = os.Getenv("PORT")
	githubOrg = os.Getenv("GITHUB_ORG")
	githubRepo = os.Getenv("GITHUB_REPO")
	toolsGithubTokenPath = os.Getenv("TOOLS_GITHUB_TOKEN_PATH")
```

Or find different name for the env name. Let's try to keep the names short as possible. Maybe `GHE_TOKEN_PATH`, which translates to "GitHub Enterprise Token Path".

**Review by Ressetkk — COMMENTED** [review]
Im also having some concerns about the code in these 2 functions. IMO initialization code (the one that is in the main() func for 2 apps) should be extracted as external package. With this we'd minimize potential growing technical debt. Usually copy-pasting the same things is not a good idea, especially if you plan on maintaining it.

Let's keep the discussion up and keep an eye on this code.

File: development/gcp/cloud-run/search-github-issue/search-github-issues.go (hunk 1 above)
Comment (dekiel) [reply development/gcp/cloud-run/search-github-issue/search-github-issues.go:58]: You can't use PORT env var as it's collides with cloud run default env vars. You will get error when starting cloud run instance with such env definition.
I don't see anything wrong with longer variable names. Using shortcuts like GHE makes them harder to understand after some time. Longer names provide more information about what this variable is.

**Comment by dekiel** [comment]
> Im also having some concerns about the code in these 2 functions. IMO initialization code (the one that is in the main() func for 2 apps) should be extracted as external package. With this we'd minimize potential growing technical debt. Usually copy-pasting the same things is not a good idea, especially if you plan on maintaining it.
> 
> Let's keep the discussion up and keep an eye on this code.

This was intentionally left to be done later to bring first working version earlier. An epic issue already has task to refactor code end extract all duplicated parts in to separate package.

**Comment by dekiel** [comment]
Env var names aligned.

**Review by halamix2 — APPROVED** [review]

**Pull request merged**